manager/install_cortex.sh

```diff
@@ -171,9 +173,16 @@ setup_configmap
 setup_secrets
 
 envsubst < manifests/spark.yaml | kubectl apply -f - >/dev/null
-envsubst < manifests/nginx.yaml | kubectl apply -f - >/dev/null
 envsubst < manifests/fluentd.yaml | kubectl apply -f - >/dev/null
+
+kubectl create namespace istio-system
+kubectl create -n istio-system secret tls istio-customgateway-certs --key cortex.example.com/3_application/private/cortex.example.com.key.pem --cert cortex.example.com/3_application/certs/cortex.example.com.cert.pem
+helm template manifests/istio-init --name istio-init --namespace istio-system | kubectl apply -f -
+sleep 20
```

Comment:
is 20 seconds safe? Is there a straightforward thing we could check in an infinite loop and continue when that's ready?